src/main.js

```diff
@@ -47,33 +61,32 @@ const zipFunction = async function (
 
   const pluginsModulesPath =
     defaultModulesPath === undefined ? await getPluginsModulesPath(srcPath) : defaultModulesPath
-  const srcFiles = await getSrcFiles({
-    runtime,
-    stat,
-    mainFile,
-    extension,
-    srcPath,
-    srcDir,
-    pluginsModulesPath,
-    useEsbuild,
-    externalModules,
-  })
 
   await makeDir(destFolder)
 
-  const destPath = await RUNTIMES[runtime]({
+  // If the file is a zip, we assume the function is bundled and ready to go.
+  // We assume its runtime to be JavaScript and simply copy it to the
+  // destination path with no further processing.
+  if (extension === '.zip') {
+    const destPath = join(destFolder, filename)
+    await cpFile(srcPath, destPath)
+    return { path: destPath, runtime: 'js' }
```

Comment:
As discussed with @ehmicky, the `runtime` property here should probably be changed to `undefined`. To be done in a separate PR.